File: lab06/echon.py
#!/usr/bin/python

import sys;


if len(sys.argv) !=3:
    print "Usage: ./echon.py <number of lines> <string>"
    sys.exit()
cishu=sys.argv[1];
zifu=sys.argv[2];
if cishu.isdigit():
    shu=int(cishu)
    while shu>0:
        print(zifu)
        shu-=1
else:
    print "./echon.py: argument 1 must be a non-negative integer"
    sys.exit() 

File: lab05/echon.pl
#!/usr/bin/perl

$size= @ARGV;
if ($size!=2){
print "Usage: ./echon.pl <number of lines> <string>\n";
exit;
}

if (@ARGV[0]=~ /^\d+$/ ){
for ($a=0; $a< @ARGV[0]; $a=$a+1){
print "@ARGV[1]\n";
}}
else{
print "./echon.pl: argument 1 must be a non-negative integer\n";
exit;
}

